stories/__tests__/SQFormDropdown.stories.test.js

```diff
@@ -138,3 +151,27 @@ it('should highlight field if required but no value selected', () => {
   expect(required).toBeVisible();
   expect(required).toHaveClass('Mui-error');
 });
+
+/* Can't test these as there's no way to pass in a different options list
```

Comment:
I haven't been able to figure out a way to make this work yet. This would require passing the options array to the `children` prop in the story, but for some reason that always gives an error saying `children` is undefined